### PR TITLE
Implementing verification with md5 hash

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#49

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -117,6 +117,7 @@
         <script src="bower_components/angularLocalStorage/src/angularLocalStorage.js"></script>
         <script src="bower_components/angular-audio-player/src/angular-audio-player.js"></script>
         <script src="bower_components/angular-audio-player/src/helpers.js"></script>
+        <script src="bower_components/angular-md5/angular-md5.js"></script>
```
Comment (amitaibu) [inline app/index.html:120]: Shouldn't `bower.json` be also updated?

File: app/scripts/services/marker.js [hunk 2]
```diff
@@ -72,9 +72,8 @@ angular.module('publicEducationApp')
           location: location
         };
 
-        // @todo Crossbrowser md5 version, require research.
-        // var hash = Crypto.md5(newMarker);
-        newMarker.hash = new Date().getTime();
+        // Creting hash.
+        newMarker.hash = md5.createHash(id);
```
Comment (amitaibu) [inline app/scripts/services/marker.js]: the `id` isn't unique -- maybe hash the entire `marker`?

File: app/index.html (hunk 1 above)
Comment (ceoaliongroo) [inline app/index.html:120]: Yes, is updated in the bower.json

**Comment by ceoaliongroo** [comment]
Corrected.

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**